dvc/command/plot.py

```diff
@@ -0,0 +1,243 @@
+                embed=not self.args.no_html,
+                csv_header=not self.args.no_csv_header,
+                title=self.args.title,
+                x_title=self.args.xlab,
+                y_title=self.args.ylab,
+            )
+
+            if self.args.stdout:
+                logger.info(plot_string)
+            else:
+                result_path = self._result_file()
+                with open(result_path, "w") as fobj:
+                    fobj.write(plot_string)
+
+                logger.info(
+                    "file://{}".format(
+                        os.path.join(self.repo.root_dir, result_path)
+                    )
+                )
+
+        except DvcException:
+            logger.exception("")
+            return 1
+
+        return 0
+
+
+class CmdPlotShow(CmdPLot):
+    def _revisions(self):
+        return None
+
+
+class CmdPlotDiff(CmdPLot):
+    def _revisions(self):
+        revisions = self.args.revisions or []
+        if len(revisions) <= 1:
+            if len(revisions) == 0 and self.repo.scm.is_dirty():
+                revisions.append("HEAD")
+            revisions.append(WORKSPACE_REVISION_NAME)
+        return revisions
+
+
+def add_parser(subparsers, parent_parser):
+    PLOT_HELP = (
+        "Generating plots for continuous metrics stored in structured files "
+        "(JSON, CSV, TSV)."
+    )
+
+    plot_parser = subparsers.add_parser(
+        "plot",
+        parents=[parent_parser],
+        description=append_doc_link(PLOT_HELP, "plot"),
+        help=PLOT_HELP,
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+    )
+    plot_subparsers = plot_parser.add_subparsers(
+        dest="cmd",
+        help="Use `dvc plot CMD --help` to display command-specific help.",
+    )
+
+    fix_subparsers(plot_subparsers)
+
+    SHOW_HELP = "Generate a plot image file from a continuous metrics file."
+    plot_show_parser = plot_subparsers.add_parser(
+        "show",
+        parents=[parent_parser],
+        description=append_doc_link(SHOW_HELP, "plot/show"),
+        help=SHOW_HELP,
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+    )
+    plot_show_parser.add_argument(
+        "-t",
+        "--template",
+        nargs="?",
+        default=None,
+        help="File to be injected with data.",
+    )
+    plot_show_parser.add_argument(
+        "-f", "--file", default=None, help="Name of the generated file."
+    )
+    plot_show_parser.add_argument(
+        "-s",
+        "--select",
+        default=None,
+        help="Choose which field(s) or JSONPath to include in the plot.",
+    )
+    plot_show_parser.add_argument(
+        "-x", default=None, help="Field name for x axis."
+    )
+    plot_show_parser.add_argument(
+        "-y", default=None, help="Field name for y axis."
+    )
+    plot_show_parser.add_argument(
+        "--stdout",
+        action="store_true",
+        default=False,
+        help="Print plot specification to stdout.",
```

Comment:
```suggestion
        help="Print the plot URL to stdout.",
```
It's really a HTML file or web page, not just the Vega spec.

Reply:
Same in line 221

Reply:
I agree that it can be HTML, but it also can be JSON when used with `--no-html`, none of these cases is an URL for the plot. Maybe lets leave it as `Print plot to stdout`?

Reply:
Gotcha. How about `Print plot location`? (`to stdout` if you want too, but maybe it's obvious).

Reply:
Gotcha. How about `Print plot location`? (`to stdout` if you want too, but maybe it's obvious).

Reply:
Added my suggestion in e0b065cf (for #3759)